apps/admin-gui/src/app/shared/components/users-list/users-list.component.ts

```diff
@@ -37,13 +37,16 @@ export class UsersListComponent implements OnChanges {
   @Input()
   inDialog: boolean;
 
+  @Input()
+  inAdmin = false;
+
```

Comment:
I definitely do not want another property in this list. If you want to hide some column do it with property [hideColumns].
When you touch it I want from you to actually redesign this so that users-list take instead of hideColumns - displayColumns like it is done in ApplicationsListComponent. So when situation like this happens and you add another column you dont have to change 1549645 files to hide the new column.